movie/views.py

```diff
@@ -1,3 +1,33 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+
+from .models import Movie
+from analysis.models import Star
+
+class UserFavoriteView(View):
+    def get(self, request):
+        account_id = request.GET.get('id')
+        list_range = request.GET.get('range')
```

Comment:
```suggestion
        list_range = request.GET.get('range', LIMIT )
```
코드 상단에 LIMIT라는 상수를 지정하시고, range에 값이 없을시 대입할 기본값을 지정해주세요.